### ISSUE TITLE
Broken cs_detail in Java Bindings

### ISSUE THREAD

**Issue by ESultanik** [capstone-engine/capstone#566]
The `cs_detail` member of `_cs_insn` is always `null` for me.  The rest of the structure, including `mnemonic` and `op_str`, is correctly populated for each instruction.  (The native library was _not_ compiled in "diet" mode.)  This means that `CsInsn.operands.op` is always zero length.

Sure enough, running TestArm.java, all of the instructions print correctly (since those printouts come from `mnemonic` and `op_str`), but none of the operand printouts happen.

I am running capstone version 3.0.4 and the latest Java bindings from commit  b0c5436558dc6c82fe1de1c0018513201f591150.

**Note**: I have only tested this so far on ARM.

**To Reproduce:** Simply run `TestArm.java` and notice that it never prints out "`top_count`".

**Comment by ESultanik** [comment]
It appears to be working correctly for X86, so this may be an ARM-only issue.

**Comment by ESultanik** [comment]
I manually set `CS_OPT_DETAIL` to `CS_OPT_ON` and `CS_OPT_SKIPDATA` to `CS_OPT_OFF`, but that didn't affect the result.  In the `disasm` function in Capstone.java, I inspected the raw bytes returned from the native call immediately after `_cs_insn byref = new _cs_insn(p);`.  The `mnemonic` and `op_str` strings are correct, however, *everything* after the `op_str` is zero bytes.  That's where the pointer to the `cs_detail` struct should be, but it's null.  So unless there's a really weird bug deep within JNA, it appears as if `cs_detail` is being set to null in the native C library.
